### PR TITLE
MIG 10.0 product_multi_category

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

File: product_multi_category/README.rst [hunk 1]
```diff
@@ -38,7 +31,7 @@ help us smashing it by providing a detailed and welcomed `feedback
 <https://github.com/OCA/
 product-attribute/issues/new?body=module:%20
 product_multi_category%0Aversion:%20
-9.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+10.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline product_multi_category/README.rst]: Use short version

**Review by pedrobaeza — APPROVED** [review]
Minor comment

File: product_multi_category/README.rst (hunk 1 above)
Comment (angelmoya) [reply product_multi_category/README.rst]: What do you mean with shot version? 10 instead of 10.0 ??

Comment (pedrobaeza) [reply product_multi_category/README.rst]: Check current README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst. I mean the short link.

**Comment by angelmoya** [comment]
@pedrobaeza fixed and squashed commits

**Review by florian-dacosta — APPROVED** [review]

**Review by Cedric-Pigeon — APPROVED** [review]

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza I think this one could be merged ;-)

**Comment by Cedric-Pigeon** [comment]
@pedrobaeza Thanks but I just noticed  that setup was missing so I created a new PR https://github.com/OCA/product-attribute/pull/260

**Comment by dreambits** [comment]
I tried using this one on nightly build of odoo 10 but it failed to find **categ_id** field.
I realized that categ_id field is added by stock module and so lower priority of view in here is causing it to not find the field.
So I changed it to 17 for now but what should be the recommended priority for this view to avoid side effects with other customization ?

P.s. Tried running this module in runbot and saw that **product_template_form_view** view itself has categ_id field so maybe this has been added after recent nighly build. Need to dig deeper.

P.p.s So I found this and it may have been merged after nightly: https://github.com/odoo/odoo/commit/8e5684250bbab7e18d46bf13fb512003602a9791

**Pull request merged**